main.go

```diff
@@ -2,56 +2,52 @@ package main
 
 import (
 	"fmt"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/ldlog"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
-	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
-	"gopkg.in/launchdarkly/go-server-sdk.v5/ldcomponents"
 	"os"
 	"time"
+
+	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
 )
 
-// Set sdkKey to your LaunchDarkly SDK key before compiling
+// Set sdkKey to your LaunchDarkly SDK key.
 const sdkKey = ""
 
-// Set featureFlagKey to the feature flag key you want to evaluate
+// Set featureFlagKey to the feature flag key you want to evaluate.
 const featureFlagKey = "my-boolean-flag"
 
+func showMessage(s string) { fmt.Printf("*** %s\n\n", s) }
+
 func main() {
 	if sdkKey == "" {
-		fmt.Println("Please edit main.go to set sdkKey to your LaunchDarkly SDK key first")
+		showMessage("Please edit main.go to set sdkKey to your LaunchDarkly SDK key first")
 		os.Exit(1)
 	}
 
-	// The only custom configuration we are doing here is to reduce the amount of logging.
-	config := ld.Config{
-		Logging: ldcomponents.Logging().MinLevel(ldlog.Warn),
-	}
-
-	client, err := ld.MakeCustomClient(sdkKey, config, 5*time.Second)
-	if err != nil {
-		fmt.Println(err.Error())
+	ldClient, _ := ld.MakeClient(sdkKey, 5*time.Second)
+	if ldClient.Initialized() {
+		showMessage("SDK successfully initialized!")
+	} else {
+		showMessage("SDK failed to initialize")
+		os.Exit(1)
 	}
 
 	// Set up the user properties. This user should appear on your LaunchDarkly users dashboard
 	// soon after you run the demo.
-	user := lduser.NewUserBuilder("bob@example.com").
-		FirstName("Bob").
-		LastName("Loblaw").
-		Custom("groups", ldvalue.ArrayBuild().Add(ldvalue.String("beta_testers")).Build()).
+	user := lduser.NewUserBuilder("example-user-key").
+		Name("Sandy").
```

Comment:
Our new simpler example user. We just want to illustrate that users can have attributes, in general; details about what the available ones are, and about custom attributes, are covered in the online docs and the SDK API docs. Also, the user key is no longer an email address because we don't want to give the impression that user keys must contain PII.